### PR TITLE
chore: add css documentation for card and login styles

### PR DESCRIPTION
**Pull request by vishnuchoutapellys**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/angular/src/layout/API.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+| --clr-login-panel-line-opacity | Changes the opacity to the divider line on the login page.|
+
+
+
+
+## Class names
+
+| Class name | Description                                                    |
+| ---------- | -------------------------------------------------------------- |
+| clr-login   | Usernames and passwords are entered on a login page to access a website or app. |
```
Comment (kevinbuhmann) [inline projects/angular/src/layout/API.md:55]: There is no `.clr-login` selector in the CSS. The class names are `login-wrapper`, `login`, etc.

**Review by kevinbuhmann — CHANGES_REQUESTED** [review]

File: projects/angular/src/layout/API.md (hunk 1 above)
Comment (vishnuchoutapellys) [reply projects/angular/src/layout/API.md]: Hi @kevinbuhmann class names are Implemented as per your suggestions.

File: projects/angular/src/layout/API.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+| --clr-card-box-shadow-color | Changes the background shadow color of card. |
+| --clr-card-clickable-border-color |  Changes the background color of card when clicked.|
+| --clr-card-clickable-box-shadow-color | Changes the background shadow color of card when clicked.|
+
+
+## Class names
+
+| Class name | Description                                                    |
+| ---------- | -------------------------------------------------------------- |
+| clr-card   | It usually has a rectangle, image, title, description, and links. CSS arranges cards for design flexibility. |
```
[the remark this answers is not in the document]
Comment (vishnuchoutapellys) [reply projects/angular/src/layout/API.md]: Hi @kevinbuhmann class names are Implemented as per your suggestions.

File: projects/angular/src/layout/API.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+| ------------------------------ | ---------------------------------------------------------------------------------------- |
+| --clr-card-bg-color            | Changes the background color of card.  |
+| --clr-card-divider-color       | Changes the divider color of the card.                              |
+| --clr-card-title-color         | Changes the text color of card header.           |
+| --clr-card-title-font-weight      | Changes the thickness of card title.                        |
+| --clr-card-border-width | Changes the border width of the card. |
+| --clr-card-border-radius | Changes the border radius of card. |
+| --clr-card-border-color | Changes the border color of the card. |
+| --clr-card-box-shadow-color | Changes the background shadow color of card. |
+| --clr-card-clickable-border-color |  Changes the background color of card when clicked.|
```
Comment (Jinnie) [inline projects/angular/src/layout/API.md]: It should not be "when clicked", more like if the card is clickable. Some cards behave like buttons.

File: projects/angular/src/layout/API.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+| --clr-card-clickable-box-shadow-color | Changes the background shadow color of card when clicked.|
```
Comment (Jinnie) [inline projects/angular/src/layout/API.md]: Same here about "clickable"

File: projects/angular/src/layout/API.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+| --clr-login-title-font-weight       | Changes the title thickness of login page.                      |
+| --clr-login-title-font-family        | Changes the font style of title in login page.           |
+| --clr-login-trademark-color      | Changes the color of trademark logo.                        |
+| --clr-login-trademark-font-weight | Changes the thickness of trademark logo in login page. |
+| --clr-login-trademark-font-family | Changes the logo style of trademark icon in the login section. |
+| --clr-login-subtitle-color | Changes the text color of the login page's sub title. |
+| --clr-login-subtitle-font-weight | Changes the thickness of the login page sub title. |
+| --clr-login-subtitle-font-family | Changes the style of the login page sub title.|
+| --clr-login-background-color | Changes the background color of the login page.|
+| --clr-login-error-background-color | Changes the background color of login page's to indicate errors.|
```
Comment (Jinnie) [inline projects/angular/src/layout/API.md]: more like "background color of the error message"

File: projects/angular/src/layout/API.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+| --clr-login-error-border-radius | Changes the border radius of login page when get the errors.|
```
Comment (Jinnie) [inline projects/angular/src/layout/API.md]: Same here "error message"

File: projects/angular/src/layout/API.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+| --clr-login-panel-line-color | Changes the color of the divider line in login page .|
```
Comment (Jinnie) [inline projects/angular/src/layout/API.md]: typo: white space between page and .

File: projects/angular/src/layout/API.md (hunk 5 above)
Comment (vishnuchoutapellys) [reply projects/angular/src/layout/API.md]: Changed the description as per the suggestions.

File: projects/angular/src/layout/API.md (hunk 6 above)
Comment (vishnuchoutapellys) [reply projects/angular/src/layout/API.md]: Changed the description as per the suggestions.

File: projects/angular/src/layout/API.md (hunk 7 above)
Comment (vishnuchoutapellys) [reply projects/angular/src/layout/API.md]: fixed the typo error.

**Review by kevinbuhmann — APPROVED** [review]

**Review by kevinbuhmann — APPROVED** [review]

File: projects/angular/src/layout/API.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+| --clr-card-clickable-border-color      | Changes the background color of clickable card.                |
+| --clr-card-clickable-box-shadow-color  | Changes the background shadow color of clickable card.         |
+
+
+## Class names
+
+| Class name            | Description                                                    |
+| --------------------- | -------------------------------------------------------------- |
+| card                  |  Card is a content container for text, photos, and actions in the context of a single subject. |
+| card-title            |  A title within the header.                                                                    |
```
Comment (kevinbuhmann) [inline projects/angular/src/layout/API.md:24]: The `.card-title` does not go in the header. It goes inside a `.card-block`. I am going to fix this myself.

Comment (vishnuchoutapellys) [reply projects/angular/src/layout/API.md:24]: Okay, I appreciate you informing me.

**Review by kevinbuhmann — APPROVED** [review]

**Pull request merged**